src/queries/completion.ts

```diff
@@ -0,0 +1,40 @@
+import type { Model, Prompt } from '../types';
+
+export type CompletionOptions = {
+  model: Model;
+  prompt: Prompt;
+};
+
+export class Completion {
```

Comment:
Can we get some class docs here too ?

I personally find the name `Completion` a bit awkward, WDYT?